hathor/manager.py

```diff
@@ -863,15 +857,7 @@ def submit_block(self, blk: Block, fails_silently: bool = True) -> bool:
         if parent_hash not in tips:
             self.log.warn('submit_block(): Ignoring block: parent not a tip', blk=blk.hash_hex)
             return False
-        parent_block = self.tx_storage.get_transaction(parent_hash)
-        parent_block_metadata = parent_block.get_metadata()
         # this is the smallest weight that won't cause the score to increase, anything equal or smaller is bad
-        min_insignificant_weight = calculate_min_significant_weight(
-            parent_block_metadata.score,
-            self._settings.WEIGHT_TOL
-        )
-        if blk.weight <= min_insignificant_weight:
-            self.log.warn('submit_block(): insignificant weight? accepted anyway', blk=blk.hash_hex, weight=blk.weight)
```

Comment:
Keep `calculate_min_significant_weight` for at least one more release. Maybe something like `TODO: remove calculate_min_significant_weight in the future`.

Reply:
Done!